c_common/models/extra_monitor_support/src/extra_monitor_support.c

```diff
@@ -841,7 +841,8 @@ static void data_in_clear_router(void) {
         if (rtr_mc_get(entry_id, &router_entry) &&
                 router_entry.key != INVALID_ROUTER_ENTRY_KEY &&
                 router_entry.mask != INVALID_ROUTER_ENTRY_MASK) {
-            rtr_free(entry_id, 1);
+            rtr_mc_set(entry_id, INVALID_ROUTER_ENTRY_KEY,
```

Comment:
this claims ALL the routing table entries. which means if we ever want to add an extra entry, where there python thinks say there's 1022 entries free. it'll go via spinnman to allocate 1, and get told there's none left. coz the rest are all been allocated to be invalid. bad bad bad

Reply:
This is not true; it *overwrites* all routing table entries.  Given the method is called "data_in_clear_router", this is surely the correct thing to do, and nothing different from what "rtr_free" would do, though only if given the correct entry!

Reply:
having discussed via skype. im inclined to accept it all. if @lplana has no objections